rclpy/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ if(WIN32)
 endif()
 
 target_link_libraries(${PROJECT_NAME}
+  ${rcl_LIBRARIES}
```

Comment:
Absolutely, I moved it from the other library target and noticed the duplicate. Removed in 8076c4e56c4c142be1438c1a8f0162e9957bed4a.